portfolio/src/main/webapp/index.html

```diff
@@ -4,18 +4,106 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link href='https://fonts.googleapis.com/css?family=Sora' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Be Vietnam' rel='stylesheet'>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <script src="script.js"></script>
   </head>
+  <style>
+  </style>
   <body>
-    <div id="content">
-      <h1>Rania Rusly's Portfolio</h1>
-      <p>Hi I love milk and chocolate but do not like chocolate milk</p>
-      <p>Mostly she likes to eat food and sleep.</p>
-      <p>I copy pasted the second line of introduction from Stanley the cat's web and apparently it perfectly describes me too</p>
-      <img src="/images/profilepic.png"/>
-      <p>Click here to get a random fact of me!</p>
-      <button onclick="addRandomFact()">poof</button>
-      <div id="fact-container"></div>
-    </div>
+    <div id="home">
+
+      <!-- navigation bar -->
+      <nav class="w3-bar w3-black">
+          <a href="/" class="w3-bar-item w3-button">Home</a>
+          <a href="experience.html" class="w3-bar-item w3-button">Experience</a>
+          <a href="interest.html" class="w3-bar-item w3-button">Interest</a>
+          <a href="#" class="w3-bar-item w3-button">Gallery</a>
+          <a href="#" class="w3-bar-item w3-button">Contact</a>
```

Comment:
Not sure if you're going to implement these pages but if so, it's nice to have a TODO explaining that :)